mmv1/products/documentaiwarehouse/DocumentSchema.yaml

```diff
@@ -0,0 +1,362 @@
+                Defines the metadata for a schema property.
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'name'
+                    description: |
+                      The name of the metadata property.
+                  - !ruby/object:Api::Type::String
+                    name: 'displayName'
+                    required: true
+                    description: |
+                      The display-name for the property, used for front-end.
+                  - !ruby/object:Api::Type::Boolean
+                    name: 'isRepeatable'
+                    description: |
+                      Whether the property can have multiple values.
+                  - !ruby/object:Api::Type::Boolean
+                    name: 'isFilterable'
+                    description: |
+                      Whether the property can be filtered. If this is a sub-property, all the parent properties must be marked filterable.
+                  - !ruby/object:Api::Type::Boolean
+                    name: 'isSearchable'
+                    description: |
+                      Indicates that the property should be included in a global search.
+                  - !ruby/object:Api::Type::Boolean
+                    name: 'isMetadata'
+                    description: |
+                      Whether the property is user supplied metadata.
+                  - !ruby/object:Api::Type::Boolean
+                    name: 'isRequired'
+                    description: |
+                      Whether the property is mandatory.
+                  - !ruby/object:Api::Type::Enum
+                    name: 'retrievalImportance'
+                    description: |
+                      Stores the retrieval importance.
+                    values:
+                      - :HIGHEST
+                      - :HIGHER
+                      - :HIGH
+                      - :MEDIUM
+                      - :LOW
+                      - :LOWEST
+                    default_value: :MEDIUM
+                  - !ruby/object:Api::Type::Array
+                    name: 'schemaSources'
+                    description: |
+                      The schema source information.
+                    item_type: !ruby/object:Api::Type::NestedObject
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'name'
+                          description: |
+                            The schema name in the source.
+                        - !ruby/object:Api::Type::String
+                          name: 'processorType'
+                          required: true
+                          description: |
+                            The Doc AI processor type name.
+                  - !ruby/object:Api::Type::NestedObject
+                    name: 'integerTypeOptions'
+                    exactly_one_of:
```

Comment:
`exactly_one_of` is not working for lists and can be removed. It is a bug on Terraform https://github.com/hashicorp/terraform-plugin-sdk/issues/470